pallets/time-release/src/lib.rs

```diff
@@ -0,0 +1,426 @@
+//! # Time-Release Module
+//!
+//! This pallet is a fork of the [ORML-vesting]( [vesting](https://github.com/open-web3-stack/open-runtime-module-library/tree/master/vesting)) polkadot-v0.9.39.
+//!
+//! ## Overview
+//!
+//! Time-release module provides a means of scheduled balance lock on an account. It
+//! uses the *graded release* way, which unlocks a specific amount of balance
+//! every period of time, until all balance unlocked.
+//!
+//! ### Release Schedule
+//!
+//! The schedule of a release is described by data structure `ReleaseSchedule`:
+//! from the block number of `start`, for every `period` amount of blocks,
+//! `per_period` amount of balance would unlocked, until number of periods
+//! `period_count` reached. Note in release schedules, *time* is measured by
+//! block number. All `ReleaseSchedule`s under an account could be queried in
+//! chain state.
+//!
+//! ## Interface
+//!
+//! ### Dispatchable Functions
+//!
+//! - `transfer` - Add a new release schedule for an account.
+//! - `claim` - Claim unlocked balances.
+//! - `update_release_schedules` - Update all release schedules under an
+//!   account, `root` origin required.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::unused_unit)]
+
+use frame_support::{
+	ensure,
+	pallet_prelude::*,
+	traits::{
+		Currency, EnsureOrigin, ExistenceRequirement, Get, LockIdentifier, LockableCurrency,
+		WithdrawReasons,
+	},
+	BoundedVec,
+};
+use frame_system::{ensure_root, ensure_signed, pallet_prelude::*};
+use sp_runtime::{
+	traits::{BlockNumberProvider, CheckedAdd, Saturating, StaticLookup, Zero},
+	ArithmeticError, DispatchResult,
+};
+use sp_std::vec::Vec;
+
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
+
+mod types;
+pub use types::*;
+
+pub mod weights;
+pub use weights::*;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+pub use module::*;
+
+pub const RELEASE_LOCK_ID: LockIdentifier = *b"trelease";
```

Comment:
I know we only get 8 char here, but it does show up in the UI.... Perhaps: `Time Rel` or `timeRels`?
```suggestion
pub const RELEASE_LOCK_ID: LockIdentifier = *b"timeRels";
```